docs/reference/search/search.asciidoc

```diff
@@ -341,8 +341,8 @@ If `true`, the exact number of hits is returned at the cost of some performance.
 If `false`, the response does not include the total number of hits matching the query.
 
 `typed_keys`::
-(Optional, Boolean) If `true`, aggregation and suggester names are be prefixed
-by their respective types in the response. Defaults to `true`.
+(Optional, Boolean) If `true`, aggregation and suggester names are prefixed
+by their respective types in the response. Defaults to `false`.
```

Comment:
It defaults to false:

https://github.com/elastic/elasticsearch/blob/917f54a08f01fc6c695d022bf4634380f0ec6373/server/src/main/java/org/elasticsearch/search/aggregations/InternalAggregation.java#L224

https://github.com/elastic/elasticsearch/blob/917f54a08f01fc6c695d022bf4634380f0ec6373/server/src/main/java/org/elasticsearch/search/suggest/Suggest.java#L296

Reply:
Nice catch 👍